lib/web/routers/router.go

```diff
@@ -1,35 +1,27 @@
 package routers
 
 import (
-	"html/template"
-	"net/url"
-
 	"go-google-scraper-challenge/lib/web/controllers"
 
 	"github.com/gin-gonic/gin"
-	eztemplate "github.com/michelloworld/ez-gin-template"
 )
 
-func ComebineRoutes(engine *gin.Engine) {
-	// Register HTML renderer
-	htmlRender := eztemplate.New()
-	htmlRender.Debug = gin.IsDebugging()
-	htmlRender.TemplatesDir = "lib/web/views/"
-	htmlRender.Layout = "layouts/application"
-	htmlRender.TemplateFuncMap = template.FuncMap{
-		"isActive": isActive,
-	}
-	engine.HTMLRender = htmlRender.Init()
+var webRoutes = map[string]map[string]string{
+	"home": {
+		"index": "/",
+	},
+	"session": {
+		"new": "/signin",
+	},
+}
```

Comment:
I think it's OK for me to omit this out and place it into the router in a simple way.
Because if we declare like this I better go with the route generator with looping from this like but it's overkill this time.
If you want to keep it like this, I'm also OK with this too 🖖

Reply:
Actually, I added this as I planned to use it in another PR, it's for the `UrlFor` template function so that I can get the path for specified controller and action 😅